sandpack-client/build.js

```diff
@@ -0,0 +1,37 @@
+/* eslint-disable @typescript-eslint/no-var-requires */
+const fs = require("fs");
+const { build } = require("esbuild");
+const package = require("./package.json");
+
+const options = {
+  entryPoints: fs
+    .readdirSync("./src")
+    .filter((src) => src.endsWith(".ts"))
+    .map((e) => `src/${e}`),
+
+  bundle: true,
+  sourcemap: true,
+  define: {
+    "process.env.CODESANDBOX_ENV": `"${process.env.CODESANDBOX_ENV}"`,
+    "process.env.PACKAGE_VERSION": `"${package.version}"`,
+  },
+  external: Object.keys({
+    ...(package.dependencies || {}),
+    ...(package.devDependencies || {}),
+    ...(package.peerDependencies || {}),
+  }),
+};
+
+build({
+  ...options,
+  format: "esm",
+  outdir: "dist/esm",
+  target: "es2019",
+}).catch(() => process.exit(1));
```

Comment:
Should this catch also log the error of does esbuild send this to stderr before throwing anyway?

Reply:
`tsc` was throwing the errors anyway, but it's a good idea to log the `esbuild` error as well, as it contains more details.